lib/matplotlib/dates.py

```diff
@@ -201,6 +220,64 @@ def _get_rc_timezone():
     MO, TU, WE, TH, FR, SA, SU)
 WEEKDAYS = (MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY)
 
+# default epoch: passed to np.datetime64...
+_epoch = None
+
+
+def _reset_epoch_test_example():
+    """
+    Reset the Matplotlib date epoch so it can be set again.
+
+    Only for use in tests and examples.
+    """
+    global _epoch
+    _epoch = None
+
+
+def set_epoch(epoch):
+    """
+    Set the epoch (origin for dates) for datetime calculations.
+
+    Parameters
+    ----------
+    epoch : str
+        valid UTC date parsable by `numpy.datetime64` (do not include
+        timezone).
+
+    The default epoch is :rc:`dates.epoch` (by default 1970-01-01T00:00).
```

Comment:
If this, and the following, paragraphs are part of the description of `epoch`, then they need to be indented. If they're about the function, they should be above `Parameters`; if they're notes, they need a `Notes` section heading.

Reply:
Ooops, fixed